### DISCUSSION TITLE
Inbound firewall was deactivated after installation

### DISCUSSION THREAD

**Discussion by Knoxi-Code** [evilsocket/opensnitch#819] [General]
Hey i have a question,

after the installation I noticed that the inbound firewall was deactivated. Is that normal or a security leak ? 

I am now thinking of reinstalling my whole system.

**Comment by gustavo-iniguez-goya** [comment]
Hi @Knoxi-Code ,

Could you explain in more detail what do you mean by "inbound firewall was deactivated"?
nftables/iptables input policy was set to allow? are you using ufw, firewalld or directly nftables/iptables?

**Describe the bug**
A clear and concise description of what the bug is.

Include the following information please:
 - OpenSnitch version.
 - OS: [e.g. Debian GNU/Linux, ArchLinux, Slackware, ...]
 - Version [e.g. Buster, 10.3, 20.04]
 - Window Manager: [e.g. GNOME Shell, KDE, enlightenment, i3wm, ...]
 - Kernel version: echo $(uname -a)

Thank you!

**Comment by Knoxi-Code** [comment]
Hey @gustavo-iniguez-goya 

after feew wekks i see that in the gui, the inbound Firewall was deactivated, look at the picture.

   - OpenSnitch version: [v1.6.0-rc.4 + Gui]
   - OS: Parrot Linux (Debian Based)
   - Version 5.1.2 "Electro Ara" 
   - Window Manager: lightdm
   - Kernel version: 6.0.0-12parrot1-amd64

I was able to observe the same phenomenon under a freshly installed Debian 11 a month ago. 
Should I reinstall my parrot system, what do you think? 

Thanks! 

![DeepinBildschirmfoto_Bereich auswählen_20230126121940](https://user-images.githubusercontent.com/115695509/214823650-7194d225-d0b7-4f6c-b750-1f202ad81f48.png)

**Reply by gustavo-iniguez-goya** [reply]
hmm, those icons mean that the Firewall is enabled. Otherwise you would see something like this:

Disabled:

![image](https://user-images.githubusercontent.com/2742953/214839785-3143095f-3eb6-4ad0-8172-db23d9e68d20.png)

Enabled :

![image](https://user-images.githubusercontent.com/2742953/214839679-23f6eaa9-fbcf-4651-bc97-5bfe065b1b4c.png)


Notice that in both cases, the other icon does not change. It looks disabled on yours because of your icons theme I think. Mine is different:

![image](https://user-images.githubusercontent.com/2742953/214840109-d683611a-3429-47fa-ace7-13501348eeca.png)

**Reply by Knoxi-Code** [reply]
@gustavo-iniguez-goya  yes of course i enable the firewall after i see it was disable, but it was disable for weeks, i dident relize that the firewall was disbled after the installation

**Reply by Knoxi-Code** [reply]
I reinstall my System at the weekend and check the settings again

**Reply by gustavo-iniguez-goya** [reply]
You don't need to reinstall the OS. Or before reinstalling, take a look at the log `/var/log/opensnitchd.log` . If there's any error it should be there.

What's the value of the field Firewall in your default-config.json file? -> `grep Firewall /etc/opensnitchd/default-config.json`
